scripts/init.sh

```diff
@@ -25,7 +25,7 @@ done
 echo "Configure rp_filter loose... "
 TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 60")
 HOST_IP=$(curl -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/meta-data/local-ipv4)
-PRIMARY_IF=$(ip -4 -o a | grep "$HOST_IP" | awk '{print $2}')
+PRIMARY_IF=$(ip -4 -o a | grep "$HOST_IP/" | awk '{print $2}')
```

Comment:
I think there are still plenty of other cases where this regex match could go wrong, particularly because `.` is a meta character:

Eg: `grep 10.1.1.1/` will match `42: dummy0 inet 10.110.1.121/32` - which is clearly not the same address.

I suggest `grep -wF "$HOST_IP"`  ... or better yet, match of the layer2 address instead:
```
PRIMARY_MAC=$(curl -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/meta-data/mac)
PRIMARY_IF=$(ip -o link show | grep -F "link/ether $PRIMARY_MAC" | awk -F'[ :]+' '{print $2}')
```